integration-cli/docker_api_stats_test.go

```diff
@@ -69,3 +71,41 @@ func (s *DockerSuite) TestStoppedContainerStatsGoroutines(c *check.C) {
 		}
 	}
 }
+
+func (s *DockerSuite) TestApiNetworkStats(c *check.C) {
+	// Run container for 30 secs
```

Comment:
I don't think so.
My understanding is that a succesfull testRequires(Network) is a prerequisite for test that need external (to host) network reachability.

This test does not need that.